pkg/volume/util/util.go

```diff
@@ -96,29 +97,44 @@ func UnmountPath(mountPath string, mounter mount.Interface) error {
 // IsNotMountPoint will be called instead of IsLikelyNotMountPoint.
 // IsNotMountPoint is more expensive but properly handles bind mounts.
```

Comment:
The comment is not applicable here? Maybe explain about isCorruptedMnt in the comment?

Reply:
isCorruptedMnt  is described here:
https://github.com/linyouchong/kubernetes/blob/2249fb36d594b1e1770187c8b1a40f82bab131ae/pkg/volume/util/util.go#L112
#L97 and #L98 describes extensiveMountPointCheck parameter.